superset-frontend/spec/javascripts/sqllab/SqlEditor_spec.jsx

```diff
@@ -72,22 +78,26 @@ describe('SqlEditor', () => {
       },
     );
 
-  it('render a SqlEditorLeftBar', () => {
+  it('render a SqlEditorLeftBar', async () => {
```

Comment:
I'm not sure some tests in this file were skipped in the first place, but I wonder if they work now :P

Reply:
They don't work. I think they are really old tests. One of them is using setState but the component is a functional component now and it seems that the height logic being tested is also different.